### ISSUE TITLE
BUG: script currently not working on Linux

### ISSUE THREAD

**Issue by Guy-Adler** [gpsystem/create-geofs-plugin#98]
When running the script on Ubuntu 20.04 (after linking to it using `npm link`), the following error occurs after gathering all the information:
```
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '/tmp/__create_geofs_plugin__3luZ5J/src/index.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/tmp/__create_geofs_plugin__3luZ5J/src/index.js'
}
```

**Comment by nicolas377** [comment]
This seems to be an error that occurs when copying nested files to the temporary directory. This isn't reproduced on Windows 10 (21H2), so this might be an issue with cross-platform handling of filesystem errors.

**Comment by nicolas377** [comment]
This seems to have been always broken (git bisect was bad all the way back to 7c3b01cdd453778638a3e9c5bf347ded72c2731b). The issue did end up being with creating the directories that files were nested in. [fsExtra.outputFile](https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputFile.md) seems to fix this on Ubuntu 20.04.
